src/tools/tools/configuration.fs

```diff
@@ -244,7 +244,7 @@ module internal ConfigurationUtils =
                 )
 
     // Unpack asm binding information
-    let UnpackAsmBinding (ms : MemStream) =
+    let UnpackAsmBinding (ms : StreamBase<byte>) =
```

Comment:
How about the corresponding PackAsmBinding function, does it also need to change to use StreamBase<byte> instead of MemStream?